core/src/main/scala/io/projectglow/sql/util/WithUtils.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 The Glow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.projectglow.sql.util
```

Comment:
We already have a `WithUtils`: https://github.com/projectglow/glow/blob/master/core/src/main/scala/io/projectglow/common/WithUtils.scala

Also, prefer composition over inheritance -- `WithUtils` should be an object, not a trait.